techniques/html/H2.html

```diff
@@ -114,50 +105,55 @@ <h2>Resources</h2>
                </li>
                
             </ul>
+            <div class="note">
+               <div role="heading" class="note-title marker" aria-level="3">訳注</div>
+               <div>
+                  <p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。<a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text">HTML 5.1§4.7.5.1. Requirements for providing text to act as an alternative for images</a> を代わりに参照できる。
```

Comment:
HTML 5.1をHTMLに置き換えておきます（WCAG 2.0達成方法集もそうすべきなのですが）
```suggestion
                  <p>HTML 4.01 は Superseded Recommendation としてマークされ、廃止された仕様である。<a href="https://html.spec.whatwg.org/multipage/images.html#alt">HTML §4.8.4.4 Requirements for providing text to act as an alternative for images</a> を代わりに参照できる。
```